src/Simplex/Chat/Options/Postgres.hs

```diff
@@ -33,16 +36,32 @@ chatDbOptsP _appDir defaultDbName = do
           <> value "simplex_v1"
           <> showDefault
       )
-  pure ChatDbOpts {dbConnstr, dbSchemaPrefix}
+  dbPoolSize <-
+    option
+      auto
+      ( long "pool-size"
+          <> metavar "DB_POOL_SIZE"
+          <> help "Database connection pool size"
+          <> value 1
+          <> showDefault
+      )
+  dbCreateSchema <-
+    switch
+      ( long "create-schema"
+          <> help "Create database schema when it does not exist"
```

Comment:
should be default true, or is it so already?

Reply:
why? I think it should be false by default